client/src/pages/Voting.jsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import location_icon from '../assets/location_icon.png';
+import clock_icon from '../assets/clock_icon.png';
+import calendar_icon from '../assets/calendar_icon.png';
+import styled from 'styled-components';
+import { useParams } from 'react-router-dom';
+
+export default function Voting({ events, setVotingConfirmation }) {
+  const { _id } = useParams();
+
+  const handleOnclickBockButton = (event) => {
+    event.preventDefault();
+    setVotingConfirmation(_id, true);
+  };
+
+  const handleOnclickWurstButton = (event) => {
+    event.preventDefault();
+    setVotingConfirmation(_id, false);
+  };
+
+  const event = events.find((event) => event._id === _id);
+  if (event === undefined) {
```

Comment:
You might also write

```suggestion
  if (!event) {
```

Reply:
Done